simulations/ContestSimulation.java

```diff
@@ -0,0 +1,192 @@
+
+
+    /*********************** Team Scenario *********************************************************************
+     * This pretends to be a team, it polls the team page every 30s and randomly performs various actions
+     */
+    // Feeder for team accounts
+    AtomicInteger counter = new AtomicInteger(1);
+    Iterator<Map<String,Object>> feeder =
+        Stream.generate((Supplier<Map<String,Object>>) () -> {
+            String username = String.format("gatling%04d", counter.getAndIncrement());
+            return Collections.singletonMap("user", username);
+        }
+    ).iterator();
+
+    HttpRequestActionBuilder teamPage = http("Team Page").get("/team");
+    ChainBuilder teamSubmitC = submit("c", "test-hello.c");
+    ChainBuilder teamSubmitCpp = submit("cpp", "test-hello.c++");
+    ChainBuilder teamSubmitJava = submit("java", "test-hello.java");
+    ChainBuilder teamSubmitKotlin = submit("kt", "test-hello.kt");
+    ChainBuilder teamSubmitPython = submit("py3", "test-hello.py3");
+    ChainBuilder teamRequestClarification = requestClarification();
+    ChainBuilder teamProblems = exec(http("Team Problemset").get("/team/problems"));
+    ChainBuilder teamScoreboard = exec(http("Team Scoreboard").get("/team/scoreboard"));
+    ScenarioBuilder teamScenario = scenario("TeamScenario")
+        .feed(feeder)
+        .exec(http("Homepage").get("/"))
+        .exec(publicScoreboard) // Fetch the public scoreboard
+        .pause(1,5)               // Brief delay
+        .exec(login())            // Log in as the team
+
+        // Mimic the behavior of the team page, it autorefreshes every 30s
+        .exec(
+            poll().pollerName("teampage")
+            .every(30)
+            .exec(teamPage)
+        )
+
+        // While we're polling the teampage, lets pretend to do some actions like a real team might
+        .repeat(NUM_TEAM_ACTIONS, "i").on(
+            exec(
+                randomSwitch().on(
+                    // 20% chance of making a submission
+                    Choice.withWeight(20.0, exec(
+                        uniformRandomSwitch().on(
+                            exec(teamSubmitC),
+                            exec(teamSubmitCpp),
+                            exec(teamSubmitJava),
+                            exec(teamSubmitKotlin),
+                            exec(teamSubmitPython)
+                        )
+                    )),
+                    // 10% of requesting clarification
+                    Choice.withWeight(10.0, teamRequestClarification),
+                    // 30% of loading problems page
+                    Choice.withWeight(30.0, teamProblems),
+                    // 40% of loading scoreboard
+                    Choice.withWeight(40.0, teamScoreboard)
+                )
+            )
+            // Sleep anywhere between 5s and 1 minute before doing another action
+            .pause(5,60)
```

Comment:
How much time between each team "action". This is in seconds, so teams will take an action between 5 and 60s after their last action.